package.json

```diff
@@ -38,7 +39,7 @@
     "tslib": "^2.3.0"
   },
   "resolutions": {
-    "rxjs": "6.6.3"
+    "rxjs": "^6.6.3"
```

Comment:
I'm curious if this was necessary or just happened?

Reply:
I think I did that to fix a warning while I was debugging things, but I don't think it's strictly necessary

Reply:
(undid it to reduce the change)